go/prometheusexporter/collector/collector.go

```diff
@@ -456,6 +466,17 @@ func (c *Collector) registerTransactionsTotalVec() {
 	c.registry.MustRegister(c.applicationMetrics.transactionsTotalVec)
 }
 
+func (c *Collector) registerExceptionsTotalVec() {
+	c.applicationMetrics.exceptionsTotalVec = prometheus.NewCounterVec(
```

Comment:
```suggestion
	c.actionMetrics.exceptionsTotalVec = prometheus.NewCounterVec(
```


I think this should be in the `actionMetrics` as it contains the `action` label.